message_definitions/v1.0/common.xml

```diff
@@ -373,6 +373,23 @@
         <description>Velocity limiting active to prevent breach</description>
       </entry>
     </enum>
+    <enum name="FENCE_TYPE">
```

Comment:
should be <enum name="FENCE_TYPE" bitmask="true">

Reply:
Fixed